packages/jsii-pacmak/lib/targets/java.ts

```diff
@@ -1049,18 +1056,21 @@ class JavaGenerator extends Generator {
     private toNativeFqn(fqn: string): string {
         const [mod, ...name] = fqn.split('.');
         const depMod = this.findModule(mod);
-        const javaPackage = depMod.targets && depMod.targets.java && depMod.targets.java.package;
-        if (!javaPackage) { throw new Error(`The module ${mod} does not have a java.package setting`); }
-
-        // since this type was needed for some reason when we generate this code, we want to make
-        // sure it's module is included as a direct dependency, even if it's not defined as a direct
-        // dependency of this assembly in jsii. this can happen, for example, when we generate
-        // interface proxies and builders which "flatten" the hirarchy.
+        // Make sure any type that's used from a dependency is listed as a direct dependency of the POM.
```

Comment:
Instead of "that's used" I would write "which is explicitly referenced"

Reply:
Makes sense. `used` is very broad.